server/commands/project_locker.go

```diff
@@ -0,0 +1,74 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+)
+
+// ProjectCommandLocker implements project locks.
+type ProjectCommandLocker struct {
```

Comment:
```suggestion
type SyncRunner struct {
```

Don't have to name it that but i think given that this is a decorator, the name should still imply that it implements `Runner`.  Sync is an easy way to say that there is some form of access constraints/locking. What do you think?